.gitignore

```diff
@@ -15,7 +15,10 @@
 /log/*.log
 /tmp
 Gemfile.lock
-Gemfile.lock
 coverage
 pkg
 spec/internal
+
+# Ignore software specific files
```

Comment:
These changes should not get committed. You should add these to a global gitignore. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore